app/services/proxies_for_owner_and_watcher_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ProxiesForOwnerAndWatcherService
```

Comment:
this sounds like a perfect candidate for a query object.. these kinds of classes should be separated from the business/service logic.. good example in here: https://mkdev.me/en/posts/how-to-use-query-objects-to-refactor-rails-sql-queries

Reply:
Makes sense! 👍🏼 Thanks for catching it! 👏🏼

Reply:
I was doubting about this, and still am, because most of it is in Ruby 🤔  I will do the commit and see the difference here 👍🏼